CHANGELOG.md

```diff
@@ -6,14 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.0.0]
+### Changed
+- **BREAKING**: The addNewKeyring method now expects an object containing the property `privateKeys` of type `string[]` in case the supplied keyring is a "Simple Keyring". ([#202](https://github.com/MetaMask/KeyringController/pull/202))
```

Comment:
Good call on highlighting this as a separate change from the TypeScript migration, but that migration seems noteworthy on its own. The package has types now!

```suggestion
- **BREAKING**: The addNewKeyring method now expects an object containing the property `privateKeys` of type `string[]` in case the supplied keyring is a "Simple Keyring". ([#202](https://github.com/MetaMask/KeyringController/pull/202))
- Migrate the project to TypeScript ([#202](https://github.com/MetaMask/KeyringController/pull/202))
  - Methods that started with an underscore are now `# private methods
  - Additional validation has been added to most methods
```